frontend/src/components/GoesData/ImageViewer.tsx

```
@@ -121,18 +121,25 @@ export default function ImageViewer({ frame, frames, onClose, onNavigate }: Imag
-        <img
-          src={`/api/goes/frames/${frame.id}/image`}
-          alt={`${frame.satellite} ${frame.band}`}
-          className="max-h-full max-w-full select-none"
-          style={{
-            transform: `scale(${scale}) translate(${translate.x / scale}px, ${translate.y / scale}px)`,
-            cursor: scale > 1 ? (dragging ? 'grabbing' : 'grab') : 'default',
-          }}
-          draggable={false}
+        <div
+          role="application"
+          tabIndex={0}
```

Biome (2.3.14)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)